nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EncodeContent.java

```diff
@@ -112,9 +153,14 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
-        final boolean encode = context.getProperty(MODE).getValue().equalsIgnoreCase(ENCODE_MODE);
-        final String encoding = context.getProperty(ENCODING).getValue();
-        final StreamCallback callback = getStreamCallback(encode, encoding);
+        final boolean encode = context.getProperty(MODE).getValue().equals(EncodingMode.ENCODE.getValue());
+               final EncodingType encoding = EncodingType.valueOf(context.getProperty(ENCODING).getValue());
```

Comment:
Please align the corrected variable with the others
```suggestion
        final boolean encode = context.getProperty(MODE).getValue().equals(EncodingMode.ENCODE.getValue());
        final EncodingType encoding = EncodingType.valueOf(context.getProperty(ENCODING).getValue());
```